### ISSUE TITLE
tmb returned with getFileCallback is missing url after upload

### ISSUE THREAD

**Issue by billynoah** [Studio-42/elFinder#3292]
I'm having an issue that only seems to arise when a custom path is set.  After uploading a new file, the thumb is visible and everything appears fine.  But when selecting the file, the object returned by `getFileCallback()` contains only the *filename* and is missing the `tmbURL`.  This *only* happens immediately after uploading and everything is fine if I change views or close/open the elfinder modal.

It also *only* seems to happen when I am using a custom path.  For example, this works fine:

```
URL: "https://example.com/image/catalog"
path: "/public_html/image/catalog"
tmbPath: "/public_html/image/cache/_elf_tmb"
tmbURL: "https://example.com/image/cache/_elf_tmb"
tmpPath: "/public_html/image/cache/.tmp"
```

But with these settings, the thumbnail url is missing after upload (the filename/hash is returned but sans url):

```
URL: "https://example.com/image/catalog/user_a"
path: "/public_html/image/catalog/user_a"
tmbPath: "/public_html/image/cache/_elf_tmb"
tmbURL: "https://example.com/image/cache/_elf_tmb"
tmpPath: "/public_html/image/cache/.tmp"
```

Is this a bug or a user error with my settings?

**Comment by nao-pon** [comment]
@billynoah It may be related to the #3292 issue. I will start the investigation.

**Comment by nao-pon** [comment]
@billynoah I fixed the problem with #3216. This issue may also have been fixed.

Can you see if the [nightly build](https://github.com/Studio-42/elFinder/archive/2.1.zip) fixes this issue?

- https://github.com/Studio-42/elFinder#downloads
